ExampleMod/Common/GlobalNPCs/EmotePickerGlobalNPC.cs

```diff
@@ -0,0 +1,28 @@
+﻿using ExampleMod.Common.Systems;
+using ExampleMod.Content.Biomes;
+using ExampleMod.Content.EmoteBubbles;
+using System.Collections.Generic;
+using Terraria;
+using Terraria.GameContent.UI;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.GlobalNPCs
+{
+	// This is where we add our regular emotes for all NPCs
+	public class EmotePickerGlobalNPC : GlobalNPC
+	{
+		public override int PickEmote(NPC npc, Player closetPlayer, List<int> emoteList, WorldUIAnchor otherAnchor) {
+			// Add the biome emote to the list if the player is in Example Biome
+			// And with random chance, this emote will be less likely to appear
+			if (Main.rand.NextBool(2) && ModContent.GetInstance<ExampleSurfaceBiome>().IsBiomeActive(closetPlayer))
+				emoteList.Add(ModContent.EmoteBubbleType<ExampleBiomeEmote>());
+
```

Comment:
This and every other single-line block statement (**in ExampleMod only!**): Change to 
```
condition {
    code
}
```